schema-templates/basic-auth.sql

```diff
@@ -0,0 +1,291 @@
+-------------------------------------------------------------------------------
+-- Adapted from https://github.com/robconery/pg-auth
+
+begin;
+
+create extension if not exists pgcrypto;
+create extension if not exists "uuid-ossp";
+create schema if not exists basic_auth;
```

Comment:
I'll add a comment because it is intentional and kind of subtle why there are two schemas. As you see the basic_auth one is meant to be hidden, with everything going through the view and few functions that are declared in the "current" (non-basic_auth) schema.